src/main/java/net/snowflake/client/core/ColumnTypeHelper.java

```diff
@@ -0,0 +1,23 @@
+package net.snowflake.client.core;
+
+import java.sql.Types;
+import net.snowflake.client.jdbc.SnowflakeUtil;
+
+public class ColumnTypeHelper {
```

Comment:
consider using `@SnowflakeJdbcInternalApi` in newly added public elements

Reply:
ok